src/test/java/net/snowflake/client/jdbc/PreparedStatementFeatureNotSupportedIT.java

```diff
@@ -14,8 +14,8 @@
 public class PreparedStatementFeatureNotSupportedIT extends BaseJDBCTest {
   @Test
   public void testFeatureNotSupportedException() throws Throwable {
-    try (Connection connection = getConnection();
-        PreparedStatement preparedStatement = connection.prepareStatement("select ?")) {
+    try (Connection connection = getConnection()) {
+      PreparedStatement preparedStatement = connection.prepareStatement("select ?");
```

Comment:
Why this change? Shouldn't we close prepared statement?

Reply:
Fixed